packages/core/src/config/recommended.ts

```diff
@@ -110,7 +110,17 @@ const recommended: PluginStyleguideConfig<'built-in'> = {
   },
   arazzoRules: {
     spec: 'error',
-    'parameters-no-body-inside-in': 'off',
+    'parameters-no-body-inside-in': 'warn',
```

Comment:
Can we update our [rulesets'](https://redocly.com/docs/cli/rules/recommended) [docs](https://redocly.com/docs/cli/rules/minimal) with the newly added rules?

Reply:
Will document in a separate PR, here is a ticket for that => https://github.com/orgs/Redocly/projects/51/views/1?pane=issue&itemId=76996901